app/modules/wps.c

```diff
@@ -0,0 +1,73 @@
+// Module for WPS
+// by F.J. Exoo
+
+#include "module.h"
+#include "lauxlib.h"
+#include "platform.h"
+
+static int wps_callback_ref;
+
+// Lua: wps.disable()
+static int ICACHE_FLASH_ATTR wps_disable(lua_State* L)
+{
+  wifi_wps_disable();
```

Comment:
I think there should be error handler, as far as wps-related functions (in includes) returns bool as success/failure status.